### PR TITLE
Linux/Mac needs sudo

### PR DESCRIPTION
**Pull request by maestroi**
Short Description: 
added sudo to installation guide
Fixes:
- sudo added for linux installation
- sudo added for Mac OSX installation

Sudo is needed to install some requirements

### CODE REVIEW

**Comment by widnyana** [comment]
@maestroi not needed if you run it inside virtualenv,

instead of messing with your system-wide package I'd prefer using virtualenv.

**Comment by maestroi** [comment]
@widnyana than maybe its good to add that to the linux instruction aswel ?

File: README.md [hunk 1]
```diff
@@ -90,20 +90,22 @@ Please keep in mind that master is not always up-to-date whereas 'dev' is. In th
...
+Make sure you install the following first:
+[Requirements](https://github.com/PokemonGoF/PokemonGo-Bot/wiki/Installation)
 
 ```
 $ git clone -b master https://github.com/PokemonGoF/PokemonGo-Bot  
 $ cd PokemonGo-Bot  
 $ virtualenv .  
 $ source bin/activate  
-$ pip install -r requirements.txt
+$ sudp pip install -r requirements.txt
```
Comment (pulgalipe) [inline README.md]: You mistyped here, it's **sudo** not _sudp_

**Comment by pulgalipe** [comment]
Agreed that using virtualenv is a safe choice, it lowers the risk of screw up your OS.
If someone would want to use multiple instances of bot by a script just make use of . notation from virtualenv.

Example:

```. ./bin/activate```

**Comment by CapCap** [comment]
Linux/mac do NOT need sudo for running it- and it's a very bad idea to tell people they need it.

**Comment by evrardjp** [comment]
👎 virtualenv is good enough.
The only change we could do is replace "(Optional)" by something like "(Recommended - optional)" in the docs

**Comment by maestroi** [comment]
Changed the instructions to use virtualenv :+1:

**Comment by douglascamata** [comment]
@maestroi please explain in the readme why virtualenv is required and it's happily merge it.

**Comment by AnAkkk** [comment]
There is no need to use virtualenv or sudo on Linux. You should pass --user if you want to install dependencies for your user, like this:
pip install -r requirements.txt --user

**Comment by douglascamata** [comment]
@AnAkkk virtualenv is not only about installing packages for the local user. It's about having many clean Python environments and being able to switch between them. Virtualenv is so useful that in Python 3.5 they embedded the idea into a `venv` module of the standard library.

**Comment by DavidK1m** [comment]
Wouldn't "Recommended" sound better than "Recommend"?

**Comment by maestroi** [comment]
@douglascamata its done i guess

**Comment by douglascamata** [comment]
you have my +1

**Comment by maestroi** [comment]
@douglascamata can you change the label ?

**Comment by MaxLeiter** [comment]
:+1: Thanks!

**Pull request merged**